frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/index.tsx

```diff
@@ -109,8 +120,8 @@ function QueryBuilderSearch({
 			notFoundContent={isFetching ? <Spin size="small" /> : null}
 		>
 			{options?.map((option) => (
-				<Select.Option key={option.value} value={option.value}>
-					{option.value}
+				<Select.Option key={option.value} value={option.label}>
```

Comment:
don't you think this will break while using value ?

Reply:
We are using Label not value @palashgdev.